scripts/build-js.js

```diff
@@ -0,0 +1,25 @@
+const browserify = require('browserify')
+const envify = require('loose-envify/custom')
+
+const buildJs = ({ basedir, debug, fakeDataPath, scriptPath, beforeBundle, env = {} }) => {
+  const b = browserify({
+    basedir,
+    debug,
+    noParse: [fakeDataPath]
+  })
+
+  if (beforeBundle) {
+    beforeBundle(b)
+  }
+
+  return b
+    .add(scriptPath)
+    .transform('brfs')
```

Comment:
browserify resolves string transform names from the `basedir`, in our case, the tool's package directory. brfs isn't necessarily installed there, even if it will usually end up there because of npm deduping.

best to `const brfs = require('brfs')` above, and then use it here.

Reply:
🙏 and that explains the error I was getting... Cheers @goto-bus-stop 👍